lib/modules/hosts/derpibooru.js

```diff
@@ -0,0 +1,216 @@
+addLibrary('mediaHosts', 'derpibooru', {
+	name: 'Derpibooru',
+	domains: [
+		'derpiboo.ru',
+		'www.derpiboo.ru',
+		'derpibooru.org',
+		'www.derpibooru.org',
+		'trixiebooru.org',
+		'www.trixiebooru.org'
```

Comment:
Ah, okay. Yeah, definitely mention that in `example.js`; better documentation will help prevent others from making the same mistake in the future.